packages/json/package.json

```diff
@@ -0,0 +1,64 @@
+{
+  "name": "@rollup/plugin-json",
+  "version": "4.0.0",
+  "publishConfig": {
+    "access": "public"
+  },
+  "description": "Convert .json files to ES6 modules",
+  "license": "MIT",
+  "repository": "rollup/plugins",
+  "main": "dist/index.js",
+  "scripts": {
+    "build": "rollup -c",
+    "ci:coverage": "nyc pnpm run test && nyc report --reporter=text-lcov > coverage.lcov",
+    "ci:lint": "pnpm run build && pnpm run lint && pnpm run security",
+    "ci:lint:commits": "commitlint --from=${CIRCLE_BRANCH} --to=${CIRCLE_SHA1}",
+    "ci:test": "pnpm run test -- --verbose && pnpm run test:ts",
+    "lint": "pnpm run lint:js && pnpm run lint:docs && pnpm run lint:package",
+    "lint:docs": "prettier --single-quote --write README.md",
+    "lint:js": "eslint --fix --cache src test",
+    "lint:package": "prettier --write package.json --plugin=prettier-plugin-package",
+    "prebuild": "del-cli dist",
+    "prepare": "npm run build",
+    "prepublishOnly": "npm run lint && npm run test",
+    "pretest": "npm run build",
+    "security": "echo 'pnpm needs `npm audit` support'",
+    "test": "ava",
+    "test:ts": "tsc index.d.ts test/types.ts --noEmit"
+  },
+  "files": [
+    "dist",
+    "index.d.ts",
+    "README.md",
+    "LICENSE"
+  ],
+  "keywords": [
+    "rollup",
+    "plugin",
+    "json",
+    "es2015",
+    "npm",
+    "modules"
+  ],
+  "peerDependencies": {
+    "rollup": "^1.20.0"
+  },
+  "dependencies": {
+    "rollup-pluginutils": "^2.5.0"
+  },
+  "devDependencies": {
+    "rollup-plugin-buble": "^0.19.6",
+    "rollup-plugin-node-resolve": "^5.2.0",
+    "source-map-support": "^0.5.11"
+  },
+  "ava": {
+    "files": [
+      "!**/fixtures/**",
+      "!**/helpers/**",
+      "!**/recipes/**",
+      "!**/types.ts"
+    ]
+  },
+  "jsnext:main": "dist/rollup-plugin-json.es.js",
```

Comment:
Why deprecated jsnext:main is still here after major release?

Reply:
it's faux-major. technically a new module, so there was no "breaking change release" when first published. The major bump was there as a signifier that this wouldn't work as a straight replacement for the old module name due to dependency changes. AFAIK rollup isn't displaying any deprecation messaging about that property, so it wasn't caught. Feel free to submit a PR to correct that. As it is, it doesn't hurt anything.